poo/polimorfismo/autoCarreras/index.ts

```diff
@@ -0,0 +1,43 @@
+class Auto{
+    private patente: string;
+    private marcaModelo: string;
+    public cambioActual: number;
+
+    public constructor(_patente:string,_marcaModelo:string,_cambioActual?:number){
+        this.patente=_patente;
+        this.marcaModelo=_marcaModelo;
+        (_cambioActual==null||_cambioActual==undefined)?this.cambioActual=1:this.cambioActual=_cambioActual;
```

Comment:
Esta linea la podes escribir asi para que quede mas claro:

`this.cambioActual = (_cambioActual==null || _cambioActual==undefined) ? 1 : _cambioActual`